packer/install-dependencies.sh

```diff
@@ -4,10 +4,14 @@ source /home/packer/provision_source.sh
 source /home/packer/cis.sh
 
 RELEASE_NOTES_FILEPATH=/var/log/azure/golden-image-install.complete
+SYSCTL_CONFIG_SRC=/home/packer/sysctl-d-60-CIS.conf
+SYSCTL_CONFIG_DEST=/etc/sysctl.d/60-CIS.conf
 
 echo "Starting build on " `date` > ${RELEASE_NOTES_FILEPATH}
 echo "Using kernel:" >> ${RELEASE_NOTES_FILEPATH}
 cat /proc/version | tee -a ${RELEASE_NOTES_FILEPATH}
+cp $SYSCTL_CONFIG_SRC $SYSCTL_CONFIG_DEST && chmod 0644 $SYSCTL_CONFIG_DEST
```

Comment:
@CecileRobertMichon can you validate that this `cp` command is doing what I think it's doing? (i.e., paving a file onto the OS fs in order to ensure the file is baked into the VHD)

Reply:
see my comment in vhd-image-builder.json

Reply:
once https://github.com/Azure/aks-engine/pull/1031 merges you can add the permissions there